hdrop-shared/src/env.rs

```diff
@@ -43,29 +43,46 @@ macro_rules! env_get {
     };
 }
 
-// Server
-env_get!(port => u16);
-env_get!(cors_origin);
-env_get!(single_file_limit_mb => usize);
-env_get!(storage_provider);
+macro_rules! env_and_const {
```

Comment:
I'd prefer a name like `map_env` because it describes an action whereas `env_and_const` doesn't, but it's really not that important. Up to you if you wanna change it or not

Reply:
That is a good suggestion. I will reconsider it later today. The term "generate" or "gen" could also be an appropriate choice.